nucliadb_node/src/services/reader.rs

```diff
@@ -385,7 +381,9 @@ impl ShardReaderService {
             .flat_map(|relation| {
                 relation
                     .unwrap()
-                    .neighbours
+                    .prefix
+                    .unwrap()
```

Comment:
Can you confirm that this `unwrap` call is safe ? If yes, can you add some comment to explain it

Reply:
It should be, as the request is a search for prefix, a `None` here would be a bug. I can't write it down or express in a different way to be more explicit about this fact

Reply:
I'll change it for an `expect`